src/applications/vaos/covid-19-vaccine/components/ContactInfoPage.jsx

```diff
@@ -46,12 +48,14 @@ const uiSchema = {
     ...phoneConfig,
     'ui:errorMessages': {
       ...phoneConfig['ui:errorMessages'],
-      pattern:
-        'Please enter a valid 10-digit phone number (with or without dashes)',
+      pattern: 'Please enter a 10-digit phone number (with or without dashes)',
```

Comment:
I don't think removing the "valid" text was intentional in the ticket. I added that the other day (after this ticket was written, I think) to reflect that we do some validation that the phone number is a real number. I think we should keep the text we already have, but might be worth checking with Lauren to be sure.

Reply:
I checked with Lauren and corrected it to read with `valid`